eng/template/azure-scaffolding.json

```diff
@@ -0,0 +1,64 @@
+{
+  "azure-core": {
+    "title": "Azure Data Plane Service",
+    "description": "Create a project representing an Azure service Data Plane API",
+    "libraries": ["@azure-tools/cadl-autorest"],
```

Comment:
Based on our other design change I suspect you don't need libraries here or below any more.